week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,43 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {if (number < 0) {
+  return true
```

Comment:
From what you've learnt during the rest of your exercises, can you think of a way to improve this function?

Reply:
Thanks for the feedback! This has been revised and made much smaller based on the predicates exercise. Let me know if you meant something else!